test/cross-browser/report.js

```diff
@@ -141,29 +160,40 @@ var createRcSocket = function() {
     return driver.executeScript('return window.createRcSocket();');
 };
 
+var clearLocalStorage = function() {
+    return driver
+        .executeScript('localStorage.clear();');
+};
+
+var synchronizeTimestamps = function() {
+    return driver
+        .executeScript('return Date.now();')
+        .then(function(d) {
+            tsOffset = Date.now() - d;
+
+            console.log('## Timestamp Offset ' + tsOffset);
+        });
+};
+
 var dumpClientLogger = function() {
-    return driver.sleep(200).then(function(){
-        return getChildWindow()
-            .then(function() {
-                return driver.executeScript('return window.clientLogger.buffer;');
-            })
-            .then(function(z) {
-                var allLogs = logger.concat(z);
-                allLogs.sort(function(a, b) {
-                    return a.ts - b.ts;
-                });
-
-                allLogs.map(function(log) {
-                    var d = '[' + new Date(log.ts).toISOString() + ']';
-                    var m = log.message.split(' ');
-
-                    var service = String('    ' + m[0]).slice(-9);
-                    m.shift();
-                    console.log(d + service + ': ' + m.join(' '));
-                });
-            })
-            .then(getParentWindow);
-    });
+    return driver
+        .executeScript('return window.dumpLogger();')
+        .then(function(z) {
+            var allLogs = logger.concat(z);
+            allLogs.sort(function(a, b) {
```

Comment:
This is embarrassing, but I legitimately have never used the native `Array.sort` method. I didn't know it existed? Or let me rephrase, I thought it was an ES5 method like `map`... Turns out its been in the spec since day 1.